router/smoke/openshift_2/inventory-amq-interconnect-1-basic-to-router.yml

```diff
@@ -18,26 +18,15 @@ all:
         router_config: /etc/qpid-dispatch/qdrouterd.conf
         implementation: dispatch
         service: qdrouterd
+        ansible_connection: local
     clients:
       hosts:
-        cli-java-0:
```

Comment:
Removed the -0 and -1 for clients, because since we are using docker, we expect that the node represents the running containers. I understood that the idea was to represent the same container multiple times in order to have different URLs on each, but if you look into conftest.py, I am doing it there, which I believe, makes it simpler to maintain.